src/index.js

```diff
@@ -0,0 +1,26 @@
+import './style.scss';
+import './style.css';
+
+import renderMeal from './modules/render.js';
+import renderLike from './modules/likeRender.js';
+import { fetchLikes } from './modules/likeAPI.js';
+
+const mainAPI = 'https://www.themealdb.com/api/json/v1/1/filter.php?c=Seafood';
+const itemCounter = document.querySelector('#counter');
+window.addEventListener('DOMContentLoaded', async () => {
+  const likes = fetchLikes();
+  fetch(mainAPI)
+    .then((response) => response.json())
+    .then((data) => {
+      const slicedData = data.meals.slice(0, 24);
+
+      slicedData.forEach((list) => {
+        renderMeal(list);
+        renderLike(list.idMeal, likes);
+      });
+      itemCounter.innerHTML = `Food(${slicedData.length})`;
```

Comment:
- [ ] You did a great job on almost all functionalities. :+1: Let's refactor the counter to achieve the project requirements. Here you are using the incoming data from the API to render the number of items. According to the requirement, we should create a function to count the number of items displayed on the screen. Kindly create a function to count the HTML nodes displayed on the screen and you can use the output of the function to render the number of foods displayed on the screen. ✍️
Please implement the counter function as a separate module to make it easier to cover it with the test.

**Project requirements**
![Screenshot from 2023-04-19 19-44-45](https://user-images.githubusercontent.com/17243325/233148332-209e84e5-ed26-4bd1-8e9c-e7b1936cbc82.png)